metrics_utility/test/test_extra_params_validation.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import pytest
+
+from openpyxl import load_workbook
+from pandas import read_excel
+
+from metrics_utility import prepare
+from metrics_utility.test.util import run_build_int
+
+
+prepare()
```

Comment:
```suggestion
from metrics_utility.test.util import run_build_int


```

prepare is no longer needed in test_* files (post rebase)